build-tools/src/main/resources/software/amazon/awssdk/spotbugs-suppressions.xml

```diff
@@ -67,4 +67,40 @@
         <Method name="onComplete" />
         <Bug pattern="NP_NONNULL_PARAM_VIOLATION" />
     </Match>
+
+    <!-- False positive -->
+    <Match>
+        <Class name="software.amazon.awssdk.core.ResponseBytes" />
+        <Bug pattern="EI_EXPOSE_REP2" />
+    </Match>
+
+    <!-- False positive Unconfirmed cast-->
+    <Match>
+        <Or>
+            <Class name="software.amazon.awssdk.core.client.BaseAsyncClientHandler$InterceptorCallingHttpResponseHandler" />
+            <Class name="software.amazon.awssdk.core.client.BaseAsyncClientHandler$UnmarshallingSdkHttpResponseHandler" />
+            <Class name="software.amazon.awssdk.core.internal.http.async.SyncResponseHandlerAdapter" />
```

Comment:
Are we sure all the new values are false positives?
For example, [this code ](https://github.com/aws/aws-sdk-java-v2/blob/master/core/src/main/java/software/amazon/awssdk/core/internal/http/async/SyncResponseHandlerAdapter.java#L63) seems to be unsafe

Reply:
Yes, this is false positive.

We are passing `SdkHttpFullResponse` to `headersReceived` method.

https://github.com/aws/aws-sdk-java-v2/blob/bffab640104711f43153e402d8c962e1cd021029/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/ResponseHandler.java#L68-L74